src/Type/Definition/NonNull.php

```diff
@@ -4,65 +4,24 @@
 
 namespace GraphQL\Type\Definition;
 
-use GraphQL\Utils\Utils;
-
 class NonNull extends Type implements WrappingType, OutputType, InputType
 {
-    /** @var NullableType */
+    /** @var Type&NullableType */
```

Comment:
usually you are using it inversely 
```suggestion
    /** @var NullableType&Type */
```